app-modules/service-management/tests/ServiceRequest/CreateServiceRequestTest.php

```diff
@@ -239,3 +273,167 @@
+            ServiceRequestResource::getUrl('create')
+        )->assertForbidden();
+});
+
+test('displays only service request types managed by the current user', function () {
+    $settings = app(LicenseSettings::class);
+
+    $settings->data->addons->serviceManagement = true;
+
+    $settings->save();
+
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    $team = Team::factory()->create();
+
+    $user->teams()->attach($team);
+
+    $user->refresh();
+
+    $user->givePermissionTo('service_request.view-any');
+    $user->givePermissionTo('service_request.create');
+
+    actingAs($user);
+
+    $serviceRequestTypesWithManagers = ServiceRequestType::factory()
+        ->hasAttached($team, [], 'managers')
+        ->create();
+
+    $serviceRequestTypesWithoutManagers = ServiceRequestType::factory()->create();
+
+    livewire(CreateServiceRequest::class)
+        ->assertFormFieldExists('type_id', function (Select $field) use ($serviceRequestTypesWithManagers, $serviceRequestTypesWithoutManagers): bool {
+            $options = $field->getOptions();
+
+            return in_array($serviceRequestTypesWithManagers->getKey(), array_keys($options)) &&
+                   ! in_array($serviceRequestTypesWithoutManagers->getKey(), array_keys($options));
+        });
+});
+
+test('create service requests if user is an auditor or manager of any service request type', function () {
+    $settings = app(LicenseSettings::class);
+
+    $settings->data->addons->serviceManagement = true;
+
+    $settings->save();
+
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    $team = Team::factory()->create();
+
+    $user->teams()->attach($team);
+
+    $user->refresh();
+
+    $user->givePermissionTo('service_request.view-any');
+    $user->givePermissionTo('service_request.create');
+
+    actingAs($user);
+
+    $serviceRequestTypesWithManager = ServiceRequestType::factory()->create();
+
+    $serviceRequestTypesWithManager->managers()->attach($team);
+
+    $serviceRequestTypesWithManager->save();
+
+    $request = collect(CreateServiceRequestRequestFactory::new()->create([
+        'priority_id' => ServiceRequestPriority::factory()->create([
+            'type_id' => $serviceRequestTypesWithManager->getKey(),
+        ])->getKey(),
+    ]));
+
+    livewire(CreateServiceRequest::class)
+        ->fillForm($request->toArray())
+        ->fillForm([
+            'respondent_id' => Contact::factory()->create()->getKey(),
+        ])
+        ->call('create')
+        ->assertHasNoFormErrors();
+
+    assertCount(1, ServiceRequest::all());
+
+    assertDatabaseHas(ServiceRequest::class, $request->except(['division_id', 'respondent_id', 'type_id'])->toArray());
+
+    $serviceRequest = ServiceRequest::first();
+
+    expect($serviceRequest->division->id)->toEqual($request['division_id']);
+});
+
+test('validate service requests type if user is not an auditor or manager of any service request type', function () {
```

Comment:
Only managers can create, ensure that is true in the name of this test and content of this test if present